### PR TITLE
product and review 

### PR DESCRIPTION
**Pull request by Nirajchemjong**
update product as per frontend and created review api

### CODE REVIEW

File: api/controller/reviewController.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const createReviewController = async (req, res, next) => {
+  try {
+    // const { rating, comment } = req.body;
+
+    // const ratingObj=
+    // const userId = getUser();
+
+    // user id can be send from redux or client using toolkit or logged in session i guess
+
```
Comment (ukiras123) [inline api/controller/reviewController.js]: Remove al the comments

File: api/model/review/ReviewModel.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  comment: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
```
Comment (ukiras123) [inline api/model/review/ReviewModel.js:25]: May be you are missing the product yhat you are reviewing, pelase check?

**Review by ukiras123 — CHANGES_REQUESTED** [review]

File: api/model/review/ReviewModel.js (hunk 2 above)
Comment (Nirajchemjong) [reply api/model/review/ReviewModel.js:25]: Added Product.

File: api/controller/reviewController.js (hunk 1 above)
Comment (Nirajchemjong) [reply api/controller/reviewController.js]: deleted the comments.

**Review by ukiras123 — APPROVED** [review]
Looks good

**Pull request merged**